src/modules/main/types/post.ts

```diff
@@ -0,0 +1,43 @@
+export interface IPost {
+    postId: number,
+    authorId: number,
+    directionId: number,
+    typeId: number,
+    title: string,
+    content: string,
+    status: PostStatus,
+    important: boolean,
+    tags: string,
+    createdAt: Date,
+    modifiedAt: Date
+}
+
+enum PostStatus {
+    'draft',
+    'moderation_first_sign',
+    'moderation_second_sign',
+    'published',
+    'archived'
+}
+
+export interface IPostTag{
+    postId: number,
+    tagId: number
+}
+
+export interface IPostCategory {
+    postId: number,
+    categoryId: number
+}
+
+export interface IPostType{
```

Comment:
don't need this one,
posttype should be enum

Reply:
have no constants to fill postType and Direction enums, created them by myself .....